index.js

```diff
@@ -19,7 +19,7 @@ async function fastifyRoutesStats (fastify, opts) {
         const [
           method,
           route
-        ] = e.name.split(':')[1].split('|')
+        ] = e.name.split(':', 2)[1].split('|')
```

Comment:
Counter offer?

```suggestion
        ] = e.name.split(':', 2)[1].split('|', 2)
```

I don’t even know at this point 🤣

Since it gets destructed to 2 variables, let’s enforce it

Reply:
Sorry took me a while to get back to this, makes sense!

Reply:
No worries! I know you're doing lots of improvements these days

Just trying to help 🙏